cleo_sdk/CLEO_Utils.h

```diff
@@ -474,6 +479,9 @@ namespace CLEO
     #define OPCODE_WRITE_PARAM_UINT(value) _writeParam(thread, value); \
         if (!_paramWasInt(true)) { SHOW_ERROR("Output argument #%d expected to be variable int, got %s in script %s\nScript suspended.", CLEO_GetParamsHandledCount(), CLEO::ToKindStr(_lastParamType, _lastParamArrayType), CLEO::ScriptInfoStr(thread).c_str()); return thread->Suspend(); }
 
+    #define OPCODE_WRITE_PARAM_ANY(value) _writeParam(thread, value); \
+        if (!_paramWasInt(true) && !_paramWasFloat(true)) { SHOW_ERROR("Output argument #%d expected to be simple-type variable, got %s in script %s\nScript suspended.", CLEO_GetParamsHandledCount(), CLEO::ToKindStr(_lastParamType, _lastParamArrayType), CLEO::ScriptInfoStr(thread).c_str()); return thread->Suspend(); }
```

Comment:
OPCODE_WRITE_PARAM_ANY_NUMBER

Output argument #%d expected to be an int or float variable,

Reply:
True, fixed.